js/site-search.js

```diff
@@ -44,8 +48,19 @@ bcpl.siteSearch = (($, window, constants) => {
 		const searchTerms = getSearchTerms();
 
 		if (searchTerms.length) {
```

Comment:
think you should either modfiy the getSearchTerms function to return an empty array or string if there are no search terms, or check the exists of searchTerms in the conditional

Reply:
Done.